R/optimise.R

```diff
@@ -0,0 +1,44 @@
+library(dplyr, warn.conflicts = FALSE)
+
+utils::globalVariables(c("zone", "intervention", "total_costs", "total_cases_averted", "value", "i", "j"))
```

Comment:
Tried to avoid this using `.data$` but it got very verbose, and I still had trouble eliminating all the warnings